### PR TITLE
Add a lookup plugin for AWS account attributes

### PR DESCRIPTION
**Pull request by s-hertel**
##### SUMMARY
Add a lookup plugin for AWS account attributes

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
lib/ansible/plugins/lookup/aws_account_attribute.py

##### ANSIBLE VERSION
```
2.5.0
```

### CODE REVIEW

**Comment by mattclay** [comment]
CI failure of the network test is unrelated to the PR. PEP 8 issues are valid.

**Comment by s-hertel** [comment]
Don't merge as-is, still need to remove logic for the env vars that was enabled in #33026.

**Comment by s-hertel** [comment]
ready_for_review

**Comment by s-hertel** [comment]
boto_status

File: lib/ansible/plugins/lookup/aws_account_attribute.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+EXAMPLES = """
+vars:
+  has_ec2_classic: "{{ lookup('aws_account_attribute', attribute='has-ec2-classic') }}"
+  # true | false
+
+  default_vpc_id: "{{ lookup('aws_account_attribute', attribute='default-vpc') }}"
+  # vpc-xxxxxxxx | none
+
+  account_details: "{{ lookup('aws_account_attribute', wantlist='true') }}"
+  # [{'AttributeName': 'supported-platforms', 'AttributeValues': [{'AttributeValue': 'VPC'}],...}]
```
Comment (ryansb) [inline lib/ansible/plugins/lookup/aws_account_attribute.py]: Do we want to flatten this in a nice way? Like `{'supported-platforms': ['VPC', ....]}` and such, similar to the tag-flattening we do?

**Pull request merged**